### PR TITLE
added new println

### PR DESCRIPTION
**Pull request by a-r-t**
I added a new println to the HelloWorld to make the program more interesting.

### CODE REVIEW

**Comment by rpekash** [comment]
Looks very hard to code

**Comment by pferhan** [comment]
Neat!

**Comment by JLaFrance3** [comment]
Good branch

**Comment by dbhargavQU** [comment]
Money

**Comment by cakeller2** [comment]
Looks like hard work

File: HelloWorld.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 public class HelloWorld {
     public static void main(String[] args) {
```
Comment (PeterZegarek) [inline HelloWorld.java:2]: What is public static void main

**Comment by NicholasTourony** [comment]
Hi Alex

**Comment by Taylorlea43** [comment]
YO

**Comment by asgoodman224** [comment]
Fire Code

**Comment by R-Seel02** [comment]
this branch ass

**Comment by TrippMenhall** [comment]
GR8 JUB

**Comment by a-r-t** [comment]
> Hi Alex

Nick Yo  my man!!!!

**Comment by zippyh** [comment]
hi

**Comment by b-regan11** [comment]
**vs code pets are cool**

**Comment by Sharktroid** [comment]
Why only two printlns?

**Comment by JeremyGerm** [comment]
hi

**Review by dbhargavQU — APPROVED** [review]

**Pull request merged**